src/fundus/parser/utility.py

```diff
@@ -232,11 +233,16 @@ def generic_author_parsing(
         value:      An input value representing author(s) which get parsed based on type
         split_on:   Only relevant for type(<value>) = str. If set, split <value> on <split_on>,
             else (default) split <value> on common delimiters
+        default_authors: Allows to filter out default authors like 'NewsDesk' or 'Redaktion'
```

Comment:
I think having this default parameter overloads the function. There is a utility function `apply_substitution_pattern_over_list` to substitute strings in a list of strings. I think this may be more suitable for this use case.